### ISSUE TITLE
Notification always says "Sent 0 mails". But the mails get sent!

### ISSUE THREAD

**Issue by karuppiah7890** [freeCodeCamp/Mail-for-Good#125]
![sent 0](https://user-images.githubusercontent.com/12808424/26965365-0883f832-4d13-11e7-9a92-872af6fcd4cc.png)

**Comment by AndrewGHC** [comment]
Hmm quite strange! Haven't seen this behaviour before. If you have any logs you could share that would be great, otherwise I will investigate this.

**Comment by karuppiah7890** [comment]
No, I don't have any logs. I will try again and let you know. But as far as I remember I didn't get any error logs. May be I can provide the info logs.

**Comment by arcdev1** [comment]
Can you confirm that AWS  SES is NOT still in Sandbox mode. Have you asked for and received a send limit increase from AWS? If not, you will see a blue alert at the top of AWS SES for sandbox mode. 

More info at http://docs.aws.amazon.com/ses/latest/DeveloperGuide/request-production-access.html

**Comment by karuppiah7890** [comment]
Thanks for the info @arcdev1 But if you clearly see my screen shot given above, I have a limit of 50,000 mails to send. And like I said in my issue heading, the mails are getting sent. It's just that the notifications show 0 as the number.

**Comment by arcdev1** [comment]
@karuppiah7890 yes I see that now. I should have looked more closely at the full thread. I saw the issue title and remembered having a similar issue when I first set things up that was related to AWS being in Sandbox mode. Sorry I couldn't be more helpful.

**Comment by karuppiah7890** [comment]
It's fine :) we can look into and see what went wrong.

**Comment by karuppiah7890** [comment]
@AndrewGHC When do the notifications come ? Only after sending all mails huh ? Because I think this can be possibly because my mails are getting sent a bit late sometimes, and that the values during sending are 0 in the database and that's why I get these notifications. But then, there have been times when I instantly get my mails in the mail box, but still the notifications show "Sent 0 mails"
